README.md

```diff
@@ -4,42 +4,46 @@ A small library for X window manipulation
 
 ## documentation
 
-Here is the full list of all the function you can use from `libwm`:
-
-    wm_init_xcb();
-    wm_kill_xcb();
-    wm_is_alive(wid);
-    wm_is_ignored(wid);
-    wm_is_listable(wid, mask);
-    wm_is_mapped(wid);
-    wm_get_focus();
-    wm_get_screen();
-    wm_get_windows(wid, **list);
-    wm_get_attribute(wid, attr);
-    wm_get_cursor(mode, wid, *x, *y);
-    wm_set_border(width, color, wid);
-    wm_set_focus(wid);
-    wm_set_cursor(x, y, mode);
-    wm_set_override(wid, mode);
-    wm_teleport(wid, w, h, x, y);
-    wm_move(wid, mode, x, y);
-    wm_remap(wid, mode);
-    wm_resize(wid, mode, w, h);
-    wm_restack(wid, mode);
+Here is the full list of all the functions provided by `libwm`:
+
+```c
```

Comment:
GitHub uses its own set of additional markdown syntax. (GFM - GitHub Flavoured Markdown)

Official documentation about it can be found [here](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown)

As this project is on GitHub and as the README.md is displayed as the front page of the Repo, I thought I could make the front page a little nicer looking.

Also, which Markdown specification are you referencing? 

- [CommonMark](http://commonmark.org/) allows the use of fenced code blocks like I used, which is also the first specification found when googling (or DDGing) "markdown specification"
- [The Markdown Syntax Writeup](https://daringfireball.net/projects/markdown/syntax) Another quasi specification (not as formal as CommonMark) you could have meant because it's the original writeup on John Gruber's blog after he created Markdown in collaboration with Aaron Swartz.

If you prefer the original I can revert the syntax changes to the function list.